### ISSUE TITLE
missing executables for awscli

### ISSUE THREAD

**Issue by andreineculau** [Homebrew/linuxbrew-core#21062]
# Bug report

**Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.**

- [x] ran `brew update` and can still reproduce the problem?
- [x] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [ ] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [x] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

## What you were trying to do (and why)

`brew install --verbose awscli`

## What happened (include command output)

Exit with zero code, but awscli is not linked.

<details>
  <summary>Command output</summary>
  <pre>

...
    byte-compiling /home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.44/libexec/lib/python3.8/site-packages/awscli/help.py to help.cpython-38.pyc
    running install_egg_info
    running egg_info
    creating awscli.egg-info
    writing awscli.egg-info/PKG-INFO
    writing dependency_links to awscli.egg-info/dependency_links.txt
    writing requirements to awscli.egg-info/requires.txt
    writing top-level names to awscli.egg-info/top_level.txt
    writing manifest file 'awscli.egg-info/SOURCES.txt'
    reading manifest file 'awscli.egg-info/SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    writing manifest file 'awscli.egg-info/SOURCES.txt'
    Copying awscli.egg-info to /home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.44/libexec/lib/python3.8/site-packages/awscli-2.0.44-py3.8.egg-info
    running install_scripts
    copying build/scripts-3.8/aws_bash_completer -> /home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.44/libexec/bin
    copying build/scripts-3.8/aws_zsh_completer.sh -> /home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.44/libexec/bin
    copying build/scripts-3.8/aws_legacy_completer -> /home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.44/libexec/bin
    copying build/scripts-3.8/aws -> /home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.44/libexec/bin
    copying build/scripts-3.8/aws_completer -> /home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.44/libexec/bin
    copying build/scripts-3.8/aws.cmd -> /home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.44/libexec/bin
    changing mode of /home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.44/libexec/bin/aws_bash_completer to 775
    changing mode of /home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.44/libexec/bin/aws_zsh_completer.sh to 775
    changing mode of /home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.44/libexec/bin/aws_legacy_completer to 755
    changing mode of /home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.44/libexec/bin/aws to 755
    changing mode of /home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.44/libexec/bin/aws_completer to 755
    changing mode of /home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.44/libexec/bin/aws.cmd to 775
    writing list of installed files to '/tmp/pip-record-4xacctjl/install-record.txt'
    Running setup.py install for awscli: finished with status 'done'
Successfully installed awscli-2.0.44
Removed build tracker: '/tmp/pip-req-tracker-hzimrc5j'
==> /home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.44/libexec/bin/python3 scripts/gen-ac-index --include-builtin-index
==> Cleaning
==> Finishing up
ln -s ../../Cellar/awscli/2.0.44/etc/bash_completion.d/aws_bash_completer aws_bash_completer
ln -s ../Cellar/awscli/2.0.44/bin/aws aws
ln -s ../Cellar/awscli/2.0.44/bin/aws_completer aws_completer
ln -s ../Cellar/awscli/2.0.44/bin/aws_legacy_completer aws_legacy_completer
ln -s ../Cellar/awscli/2.0.44/share/awscli awscli
ln -s ../../../Cellar/awscli/2.0.44/share/zsh/site-functions/_aws _aws
ln -s ../../../Cellar/awscli/2.0.44/share/zsh/site-functions/aws_zsh_completer.sh aws_zsh_completer.sh
==> Caveats
The "examples" directory has been installed to:
  /home/linuxbrew/.linuxbrew/share/awscli/examples

Bash completion has been installed to:
  /home/linuxbrew/.linuxbrew/etc/bash_completion.d
==> Summary
🍺  /home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.44: 13,925 files, 106.2MB, built in 3 minutes 31 seconds
==> No outdated dependents to upgrade!
==> Checking for dependents of upgraded formulae...
==> Reinstalling 1 broken dependent from source:
awscli
rm /home/linuxbrew/.linuxbrew/bin/aws
rm /home/linuxbrew/.linuxbrew/bin/aws_completer
rm /home/linuxbrew/.linuxbrew/bin/aws_legacy_completer
rm /home/linuxbrew/.linuxbrew/etc/bash_completion.d/aws_bash_completer
rm /home/linuxbrew/.linuxbrew/share/awscli
rm /home/linuxbrew/.linuxbrew/share/zsh/site-functions/_aws
rm /home/linuxbrew/.linuxbrew/share/zsh/site-functions/aws_zsh_completer.sh
==> Caveats
==> openssl@1.1
A CA file has been bootstrapped using certificates from the system
keychain. To add additional certificates, place .pem files in
  /home/linuxbrew/.linuxbrew/etc/openssl@1.1/certs

and run
  /home/linuxbrew/.linuxbrew/opt/openssl@1.1/bin/c_rehash
==> python@3.8
Python has been installed as
  /home/linuxbrew/.linuxbrew/bin/python3

Unversioned symlinks `python`, `python-config`, `pip` etc. pointing to
`python3`, `python3-config`, `pip3` etc., respectively, have been installed into
  /home/linuxbrew/.linuxbrew/opt/python@3.8/libexec/bin

You can install Python packages with
  pip3 install <package>
They will install into the site-package directory
  /home/linuxbrew/.linuxbrew/lib/python3.8/site-packages

See: https://docs.brew.sh/Homebrew-and-Python
==> awscli
The "examples" directory has been installed to:
  /home/linuxbrew/.linuxbrew/share/awscli/examples

  </pre>
</details>

Note the `==> Reinstalling 1 broken dependent from source: awscli` bit which ends up unlinking `awscli`, and then it never gets linked.

## What you expected to happen

`aws` executable to be installed.

## Step-by-step reproduction instructions (by running `brew install` commands)

```
docker run --rm -it homebrew/ubuntu18.04
brew install --verbose awscli
brew config
brew doctor
```

`brew config` shows

```
HOMEBREW_VERSION: 2.4.16
ORIGIN: https://github.com/Homebrew/brew
HEAD: 2a4c9f610e69b9fed1eec61ac2c53e4b11a32e4c
Last commit: 5 days ago
Core tap ORIGIN: https://github.com/Homebrew/linuxbrew-core
Core tap HEAD: 4221c38112e6487a59e169948ad2c0344d9bb9f7
Core tap last commit: 27 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /home/linuxbrew/.linuxbrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 4
CPU: quad-core 64-bit ivybridge
Homebrew Ruby: 2.6.3 => /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.3_2/bin/ruby
Clang: N/A
Git: 2.28.0 => /usr/bin/git
Curl: 7.58.0 => /usr/bin/curl
Kernel: Linux 4.19.76-linuxkit x86_64 GNU/Linux
OS: Ubuntu 18.04.5 LTS (bionic)
Host glibc: 2.27
/usr/bin/gcc: 7.5.0
glibc: N/A
gcc: N/A
xorg: N/A
```

`brew doctor`shows

```
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: You have unlinked kegs in your Cellar.
Leaving kegs unlinked can lead to build-trouble and cause brews that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  awscli

Warning: Broken symlinks were found. Remove them with `brew cleanup`:
  /home/linuxbrew/.linuxbrew/opt/awscli
  /home/linuxbrew/.linuxbrew/opt/awscli@2
```

**Comment by andreineculau** [comment]
I rushed to open the issue. After linking manually, invoking `aws` gives

`bash: /home/linuxbrew/.linuxbrew/bin/aws: /home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.44/libexec/bin/python3.8: bad interpreter: No such file or directory`

**Comment by andreineculau** [comment]
because the executables are in `/home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.44.reinstall/libexec/bin`

first time I encounter this behaviour

**Comment by andreineculau** [comment]
the log output was too big, so I thought I'll just paste the relevant bit. Silly me, possible another relevant bit was higher up.

```
ERROR: After October 2020 you may experience errors when installing or updating packages. This is because pip will change the way that it resolves dependency conflicts.

We recommend you use --use-feature=2020-resolver to test your packages with the new resolver before it becomes the default.

awscli 2.0.44 requires botocore==2.0.0dev48, but you'll have botocore 2.0.0.dev50 which is incompatible.
Successfully installed PyInstaller-3.5 altgraph-0.17 appdirs-1.4.4 awscli-2.0.44 botocore-2.0.0.dev50 cffi-1.14.2 colorama-0.4.3 cryptography-2.8 distlib-0.3.1 distro-1.5.0 docutils-0.15.2 filelock-3.0.12 jmespath-0.10.0 jsonschema-2.5.1 mock-1.3.0 nose-1.3.7 pbr-5.5.0 pluggy-0.13.1 prompt-toolkit-2.0.10 py-1.9.0 pycparser-2.20 python-dateutil-2.8.1 ruamel.yaml-0.15.100 s3transfer-0.3.3 setuptools-50.2.0 six-1.15.0 tox-2.9.1 urllib3-1.25.10 virtualenv-20.0.31 wcwidth-0.1.9 wheel-0.35.1
Removed build tracker: '/tmp/pip-req-tracker-2lawtzpv'
==> /home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.44/libexec/bin/pip uninstall -y awscli
Found existing installation: awscli 2.0.44
Uninstalling awscli-2.0.44:
  Successfully uninstalled awscli-2.0.44
==> /home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.44/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /tmp/awscli-20200905-3937-uiev5h/aws-cli-2.0.44
Using pip 20.2.2 from /home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.44/libexec/lib/python3.8/site-packages/pip (python 3.8)
Non-user install because user site-packages disabled
Created temporary directory: /tmp/pip-ephem-wheel-cache-1u7plmlx
Created temporary directory: /tmp/pip-req-tracker-hzimrc5j
Initialized build tracking at /tmp/pip-req-tracker-hzimrc5j
Created build tracker: /tmp/pip-req-tracker-hzimrc5j
Entered build tracker: /tmp/pip-req-tracker-hzimrc5j
Created temporary directory: /tmp/pip-install-b3qmw8v1
Processing /tmp/awscli-20200905-3937-uiev5h/aws-cli-2.0.44
```

**Comment by andreineculau** [comment]
for the time being one workaround is to uninstall and install again. Probably `brew reinstall awscli` works as well.

```
brew install awscli
which aws # no aws

brew uninstall awscli
brew install awscli
```

**Comment by jonchang** [comment]
I believe this has been fixed in 62667b011c5f166af0285955de56c4672f6b0145.

```console
% docker run --rm -it homebrew/brew
root@a8e59f4a6f50:/home/linuxbrew# export HOMEBREW_DEVELOPER=1
root@a8e59f4a6f50:/home/linuxbrew# brew install awscli
Updating Homebrew...
==> Homebrew has enabled anonymous aggregate formula and cask analytics.
Read the analytics documentation (and how to opt-out) here:
  https://docs.brew.sh/Analytics
No analytics have been recorded yet (or will be during this `brew` run).

==> Homebrew is run entirely by unpaid volunteers. Please consider donating:
  https://github.com/Homebrew/brew#donations
==> Auto-updated Homebrew!
Updated Homebrew from cd4a3c058 to e30ed4243.
Updated 1 tap (homebrew/core).
==> New Formulae
asroute                   libpciaccess              libxinerama               rustscan
cloudformation-guard      libpthread-stubs          libxkbfile                server-go
commitizen                libsm                     libxmu                    shtools
envoy                     libx11                    libxpm                    structurizr-cli
fleet-cli                 libxau                    libxrandr                 tfsec
flit                      libxaw                    libxrender                trunk
font-util                 libxaw3d                  libxres                   util-macros
git-hound                 libxcb                    libxscrnsaver             vivid
gitql                     libxcomposite             libxshmfence              webify
halide                    libxcursor                libxt                     x86_64-elf-gdb
leaf                      libxdamage                libxtst                   xcb-proto
libdmx                    libxdmcp                  libxv                     xdpyinfo
libfontenc                libxext                   libxvmc                   xorgproto
libfs                     libxfixes                 libxxf86dga               xtrans
libgccjit                 libxfont                  libxxf86vm
libhandy                  libxft                    matplotplusplus
libice                    libxi                     python@3.9
==> Updated Formulae
Updated 821 formulae.
==> Deleted Formulae
xu4

==> Downloading https://linuxbrew.bintray.com/bottles/patchelf-0.11.x86_64_linux.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/73/7396e6cf5dbe62fae0e7fcba57f88586359561824df9cfe1127b
######################################################################## 100.0%
==> Downloading https://linuxbrew.bintray.com/bottles/gdbm-1.18.1_1.x86_64_linux.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/dd/dd00a26fa20413f81477af032587de19bb620eef352a6d8dd3d9
######################################################################## 100.0%
==> Downloading https://linuxbrew.bintray.com/bottles/openssl%401.1-1.1.1g.x86_64_linux.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/64/6487415cb2902e5837927427a6b93a579bf5481652e526422d73
######################################################################## 100.0%
==> Downloading https://linuxbrew.bintray.com/bottles/ncurses-6.2.x86_64_linux.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/fc/fc5a0983a8afae8ef46945f3bec9dba9d66e81e879027d100d05
######################################################################## 100.0%
==> Downloading https://linuxbrew.bintray.com/bottles/readline-8.0.4.x86_64_linux.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/46/46da47db3da04b2f248e3cf2d6d14c55aa543555d1134f6cbbf0
######################################################################## 100.0%
==> Downloading https://linuxbrew.bintray.com/bottles/zlib-1.2.11.x86_64_linux.bottle.tar.gz
######################################################################## 100.0%
==> Downloading https://linuxbrew.bintray.com/bottles/sqlite-3.33.0.x86_64_linux.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/ba/bafd40d3bcbfe8350a04bd65fd2fced3ea98a74b27c6102972ec
######################################################################## 100.0%
==> Downloading https://linuxbrew.bintray.com/bottles/xz-5.2.5.x86_64_linux.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/67/676b0e178b3e7644f86385cb2497ac5ec490e2222ba3d0147e28
######################################################################## 100.0%
==> Downloading https://linuxbrew.bintray.com/bottles/bzip2-1.0.8.x86_64_linux.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/e5/e5fce257b9fee119d28d0e7f7dac9332bd26b248fe35d983ba85
######################################################################## 100.0%
==> Downloading https://linuxbrew.bintray.com/bottles/libffi-3.3.x86_64_linux.bottle.tar.gz
######################################################################## 100.0%
==> Downloading https://linuxbrew.bintray.com/bottles/unzip-6.0_6.x86_64_linux.bottle.tar.gz
######################################################################## 100.0%
==> Downloading https://linuxbrew.bintray.com/bottles/python%403.8-3.8.6.x86_64_linux.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/92/92944878ce148a4c168bb499d89174ed6be9ad4cf60fc1e42ecf
######################################################################## 100.0%
==> Downloading https://linuxbrew.bintray.com/bottles/groff-1.22.4.x86_64_linux.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/d3/d31274339f07f31fa1a710f04120927f11c8ad03d780ddd3e393
######################################################################## 100.0%
==> Downloading https://linuxbrew.bintray.com/bottles/libyaml-0.2.5.x86_64_linux.bottle.tar.gz
######################################################################## 100.0%
==> Downloading https://linuxbrew.bintray.com/bottles/awscli-2.0.56.x86_64_linux.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/d9/d930d656518d640ead0785c83f1f04714e43287267e85e48a20f
######################################################################## 100.0%
==> Installing dependencies for awscli: patchelf, gdbm, openssl@1.1, ncurses, readline, zlib, sqlite, xz, bzip2, libffi, unzip, python@3.8, groff and libyaml
==> Installing awscli dependency: patchelf
==> Pouring patchelf-0.11.x86_64_linux.bottle.tar.gz
🍺  /home/linuxbrew/.linuxbrew/Cellar/patchelf/0.11: 7 files, 909.2KB
==> Installing awscli dependency: gdbm
==> Pouring gdbm-1.18.1_1.x86_64_linux.bottle.tar.gz
🍺  /home/linuxbrew/.linuxbrew/Cellar/gdbm/1.18.1_1: 40 files, 1MB
==> Installing awscli dependency: openssl@1.1
==> Pouring openssl@1.1-1.1.1g.x86_64_linux.bottle.tar.gz
==> Downloading https://curl.haxx.se/ca/cacert-2020-01-01.pem
######################################################################## 100.0%
==> Caveats
A CA file has been bootstrapped using certificates from the system
keychain. To add additional certificates, place .pem files in
  /home/linuxbrew/.linuxbrew/etc/openssl@1.1/certs

and run
  /home/linuxbrew/.linuxbrew/opt/openssl@1.1/bin/c_rehash
==> Summary
🍺  /home/linuxbrew/.linuxbrew/Cellar/openssl@1.1/1.1.1g: 8,060 files, 21.4MB
==> Installing awscli dependency: ncurses
==> Pouring ncurses-6.2.x86_64_linux.bottle.tar.gz
🍺  /home/linuxbrew/.linuxbrew/Cellar/ncurses/6.2: 3,901 files, 9.5MB
==> Installing awscli dependency: readline
==> Pouring readline-8.0.4.x86_64_linux.bottle.tar.gz
🍺  /home/linuxbrew/.linuxbrew/Cellar/readline/8.0.4: 53 files, 1.9MB
==> Installing awscli dependency: zlib
==> Pouring zlib-1.2.11.x86_64_linux.bottle.tar.gz
🍺  /home/linuxbrew/.linuxbrew/Cellar/zlib/1.2.11: 12 files, 424KB
==> Installing awscli dependency: sqlite
==> Pouring sqlite-3.33.0.x86_64_linux.bottle.tar.gz
🍺  /home/linuxbrew/.linuxbrew/Cellar/sqlite/3.33.0: 12 files, 4.9MB
==> Installing awscli dependency: xz
==> Pouring xz-5.2.5.x86_64_linux.bottle.tar.gz
🍺  /home/linuxbrew/.linuxbrew/Cellar/xz/5.2.5: 128 files, 1.8MB
==> Installing awscli dependency: bzip2
==> Pouring bzip2-1.0.8.x86_64_linux.bottle.tar.gz
🍺  /home/linuxbrew/.linuxbrew/Cellar/bzip2/1.0.8: 30 files, 549.3KB
==> Installing awscli dependency: libffi
==> Pouring libffi-3.3.x86_64_linux.bottle.tar.gz
🍺  /home/linuxbrew/.linuxbrew/Cellar/libffi/3.3: 17 files, 552.7KB
==> Installing awscli dependency: unzip
==> Pouring unzip-6.0_6.x86_64_linux.bottle.tar.gz
🍺  /home/linuxbrew/.linuxbrew/Cellar/unzip/6.0_6: 17 files, 581.8KB
==> Installing awscli dependency: python@3.8
==> Pouring python@3.8-3.8.6.x86_64_linux.bottle.tar.gz
==> /home/linuxbrew/.linuxbrew/Cellar/python@3.8/3.8.6/bin/python3 -s setup.py --no-user-cfg install --
==> /home/linuxbrew/.linuxbrew/Cellar/python@3.8/3.8.6/bin/python3 -s setup.py --no-user-cfg install --
==> /home/linuxbrew/.linuxbrew/Cellar/python@3.8/3.8.6/bin/python3 -s setup.py --no-user-cfg install --
==> Caveats
Python has been installed as
  /home/linuxbrew/.linuxbrew/bin/python3

Unversioned symlinks `python`, `python-config`, `pip` etc. pointing to
`python3`, `python3-config`, `pip3` etc., respectively, have been installed into
  /home/linuxbrew/.linuxbrew/opt/python@3.8/libexec/bin

You can install Python packages with
  pip3 install <package>
They will install into the site-package directory
  /home/linuxbrew/.linuxbrew/lib/python3.8/site-packages

See: https://docs.brew.sh/Homebrew-and-Python
==> Summary
🍺  /home/linuxbrew/.linuxbrew/Cellar/python@3.8/3.8.6: 3,990 files, 73.8MB
==> Installing awscli dependency: groff
==> Pouring groff-1.22.4.x86_64_linux.bottle.tar.gz
🍺  /home/linuxbrew/.linuxbrew/Cellar/groff/1.22.4: 619 files, 21.2MB
==> Installing awscli dependency: libyaml
==> Pouring libyaml-0.2.5.x86_64_linux.bottle.tar.gz
🍺  /home/linuxbrew/.linuxbrew/Cellar/libyaml/0.2.5: 11 files, 391.3KB
==> Installing awscli
==> Pouring awscli-2.0.56.x86_64_linux.bottle.tar.gz
==> Caveats
The "examples" directory has been installed to:
  /home/linuxbrew/.linuxbrew/share/awscli/examples

Bash completion has been installed to:
  /home/linuxbrew/.linuxbrew/etc/bash_completion.d
==> Summary
🍺  /home/linuxbrew/.linuxbrew/Cellar/awscli/2.0.56: 11,494 files, 89.9MB
e==> Caveats
==> openssl@1.1
A CA file has been bootstrapped using certificates from the system
keychain. To add additional certificates, place .pem files in
  /home/linuxbrew/.linuxbrew/etc/openssl@1.1/certs

and run
  /home/linuxbrew/.linuxbrew/opt/openssl@1.1/bin/c_rehash
==> python@3.8
Python has been installed as
  /home/linuxbrew/.linuxbrew/bin/python3

Unversioned symlinks `python`, `python-config`, `pip` etc. pointing to
`python3`, `python3-config`, `pip3` etc., respectively, have been installed into
  /home/linuxbrew/.linuxbrew/opt/python@3.8/libexec/bin

You can install Python packages with
  pip3 install <package>
They will install into the site-package directory
  /home/linuxbrew/.linuxbrew/lib/python3.8/site-packages

See: https://docs.brew.sh/Homebrew-and-Python
==> awscli
The "examples" directory has been installed to:
  /home/linuxbrew/.linuxbrew/share/awscli/examples

Bash completion has been installed to:
  /home/linuxbrew/.linuxbrew/etc/bash_completion.d
root@a8e59f4a6f50:/home/linuxbrew# which aws
/home/linuxbrew/.linuxbrew/bin/aws
```
